datanode/api/trading_data_v2.go

```diff
@@ -3048,6 +3055,78 @@ func (t *tradingDataServiceV2) GetVegaTime(ctx context.Context, _ *v2.GetVegaTim
 
 // -- NetworkHistory --.
 
+func (t *tradingDataServiceV2) ExportNetworkHistory(req *v2.ExportNetworkHistoryRequest, stream v2.TradingDataService_ExportNetworkHistoryServer) error {
```

Comment:
It would be super useful if a user could give a time range, this would mean an additional index in the store by vega time and a bit of extra meta data in the segments to record the block time span.